src/js/apps/patients/worklist/worklist_state.js

```diff
@@ -32,7 +32,9 @@ export default Backbone.Model.extend({
         relativeDate: null,
       },
       isFiltering: false,
-      filters: {},
+      filters: {
+        states: {},
+      },
```

Comment:
The structure for the filters state will look like this:

```js
filters: {
  insurance: 'value',
  groupId: 'id',
  states: {
    id: true,
    id: false // will be set to false when un-selected by a user
  }
}
```

`filters.states` matches how the selected actions/flows state is built.

Its initial values are loaded from local storage. If there's no local storage item for the list, the default states will be selected initially (via the `getInitialStatesObject()` function below. Otherwise, a user's previous filter choice will be persisted on reload/route changes.